src/components/CreateNewProjectWizard/MainPane.js

```diff
@@ -191,9 +192,9 @@ class MainPane extends PureComponent<Props> {
   }
 }
 
-const Wrapper = animated(styled.div.attrs(props => ({
+const Wrapper = animated(styled.div.attrs(({ translateY }) => ({
   style: {
-    transform: `translateY(${props.translateY}px)`,
+    transform: interpolate([translateY], y => `translateY(${y}px)`),
```

Comment:
Unfortunately this interpolate change isn't producing quite the right result... it's still sticking to the top and not having the correct offset animation. The correct behavior has offset at 50 at first, and when moving to next step, offset becomes 0 and content shifts up smoothly:

<img width="200" src="https://user-images.githubusercontent.com/17421347/57203971-2389e680-6f69-11e9-8ebb-4e42f2aeea79.gif">

But I dug into it a little more and I think I found the solution! Remove the `native` prop on Spring:

```js
<Spring
  from={{
    offset: currentStepIndex === 0 ? 0 : 50,
  }}
  to={{
    offset: currentStepIndex === 0 ? 50 : 0,
  }}
  // remove
  native
>
```

And then reset this to the original code:

```js
const Wrapper = animated(styled.div.attrs(({ translateY }) => ({
  style: {
    transform: `translateY(${translateY}px)`,
  },
}))`
  height: 75vh;
  will-change: transform;
`);
```

This fixes it for me :) I'm not really sure what native is or why it's here, do you know?

Reply:
ah, OK. The native prop is there for better performance  and we should keep it. [docs](https://www.react-spring.io/docs/props/performance) about native prop.

Maybe inlining the translate into a `style` prop in MainPane could help. I'll have a look later today.

Reply:
@melanieseltzer Please have a look at the animation. It should be OK now. I'm not sure why `animation(styled.div)` is not working - maybe I'll create a minimal demo in a Codesandbox to see if it's happening there too.
Could be an issue with Styled-components or React-spring because `animation` wrapping is recommended in React-Spring docs.

Reply:
I've created a [Codesandbox](https://codesandbox.io/s/rwy86334kq)  with it - seems like the style is not added on the element but I'm not sure why.

Reply:
Nice, the updated code works 👍 Not sure why it's doing this either. I noticed some more animations not firing so I went and updated them using the way you've outlined, could you double check the logic? Feel free to modify if there's a more elegant way to write these.